### ISSUE TITLE
Write cassette: encoding bug

### ISSUE THREAD

**Issue by alex-pancho** [schemathesis/schemathesis#708]
An error occurs when trying to record a cassette:
```
Traceback (most recent call last):
  File "C:\Python\Python37\lib\threading.py", line 926, in _bootstrap_inner
    self.run()
  File "C:\Python\Python37\lib\threading.py", line 870, in run
    self._target(*self._args, **self._kwargs)
  File "d:\venv\lib\site-packages\schemathesis\cli\cassettes.py", line 140, in worker
    http_version: '{interaction.response.http_version}'"""
  File "C:\Python\Python37\lib\encodings\cp1251.py", line 19, in encode
    return codecs.charmap_encode(input,self.errors,encoding_table)[0]
UnicodeEncodeError: 'charmap' codec can't encode characters in position 460-461: character maps to <undefined>
```
maybe needed set utf-8 encoding for filewriter

**Comment by Stranger6667** [comment]
Hi @alex-pancho !

Thank you for reporting this! :) I will take a look at it during the weekend

**Comment by Stranger6667** [comment]
The fix is released in `2.3.4` (https://pypi.org/project/schemathesis/2.3.4/)

Let me know if it fixes your use case :)

**Comment by alex-pancho** [comment]
all ok, thanks
